zeppelin-web/test/karma.conf.js

```diff
@@ -18,15 +18,42 @@ module.exports = function(config) {
 
     // list of files / patterns to load in the browser
     files: [
```

Comment:
@Leemoonsoo 
'wiredep' can be improve to handle those dependencies. as index.html.
I will improve it.